msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/brokerapi/TestCase1561136.java

```diff
@@ -106,12 +109,14 @@ public void handleUserInteraction() {
 
     @Override
     public LabQuery getLabQuery() {
-        return null;
+        return LabQuery.builder()
+                .azureEnvironment(AzureEnvironment.AZURE_CLOUD)
+                .build();
     }
 
     @Override
     public TempUserType getTempUserType() {
-        return TempUserType.BASIC;
+        return null;
```

Comment:
Why changing this?

Reply:
Saves some time, as there is a wait after creating a temp user. There doesn't seem to be a strong distinction that the test requires creation of a new Temporary user (I see it says "create", but I believe we've interpreted that in the past as meaning either create a temp user or fetch an existing basic account). I can change it back if we feel this is affecting the integrity of the test.